src/features/controls.ts

```diff
@@ -327,11 +348,31 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
           // again for this particular <model-element> instance:
           this[$userPromptedOnce] = true;
           this[$waitingToPromptUser] = false;
+          this[$promptElementVisible] = true;
 
           this[$promptElement].classList.add('visible');
         }
       }
 
+      if (this[$promptElementVisible] &&
+          this.interactionPromptStyle === InteractionPromptStyle.WIGGLE) {
+        const transformString =
+            self.getComputedStyle(this[$promptAnimatedContainer])
+                .getPropertyValue('transform');
+        // Parse the fifth term of computed transform style
+        // which is in form of matrix(n0, n1, n2, n3, n4, n5)
+        // @see https://www.w3.org/TR/css-transforms-1/#serialization-of-the-computed-value
+        const offset = parseFloat(transformString.split(',')[4]);
+        const delta = offset - this[$lastPromptOffset];
+
+        if (isFinite(offset)) {
+          this[$lastPromptOffset] = offset;
+          this[$scene].pivot.rotation.y +=
```

Comment:
I think this should use the setRotation method instead. I think if you move the camera target you'll see incorrect behavior.

Reply:
Will fix